streams/src/test/java/org/apache/kafka/streams/state/internals/AbstractDualSchemaRocksDBSegmentedBytesStoreTest.java

```diff
@@ -107,7 +109,17 @@ public abstract class AbstractDualSchemaRocksDBSegmentedBytesStoreTest<S extends
     final long segmentInterval = 60_000L;
     final String storeName = "bytes-store";
 
-    @Before
+    abstract boolean hasIndex();
```

Comment:
do we need this method? it seems the `SchemaType` already declares that by the naming.